### PR TITLE
Fix the travis execution and notify Galaxy

### PR DESCRIPTION
**Pull request by danypr92**
> Squash commits in the merge

### CODE REVIEW

File: tasks/certificate.yml [hunk 1]
```diff
@@ -5,5 +5,5 @@
   register: letsencrypt_cert
 
 - name: Generate new certificate if one doesn't exist
-  shell: "certbot certonly --nginx --email '{{ letsencrypt_email }}' --agree-tos -d '{{ domain_name }}' {% if certbot_nginx_cert_name is defined %} --cert-name '{{ certbot_nginx_cert_name }}' {% endif %} {% if letsencrypt_staging %} --staging {% endif %}"
+  shell: "certbot certonly --nginx --email '{{ letsencrypt_email }}' --agree-tos -d '{{ domain_name }}' {% if certbot_nginx_cert_name is defined %} --cert-name '{{ certbot_nginx_cert_name }}' {% endif %} {% if letsencrypt_staging %} --staging {% endif %}"  # noqa 204
```
Comment (raneq) [inline tasks/certificate.yml]: What if we split the line?

```suggestion
  shell: >
    "certbot certonly --nginx --email '{{ letsencrypt_email }}'
    --agree-tos -d '{{ domain_name }}'
    {% if certbot_nginx_cert_name is defined %}
    --cert-name '{{ certbot_nginx_cert_name }}'
    {% endif %}
    {% if letsencrypt_staging %} --staging {% endif %}"
```

Would this work? Do you like it? [Source](https://robertdebock.nl/2018/11/01/lines-should-be-no-longer-than-120-chars.html)

File: .travis.yml [hunk 2]
```diff
@@ -11,3 +11,5 @@ script:
   - molecule --version
   - ansible --version
   - molecule test
+notifications:
+  webhooks: https://galaxy.ansible.com/api/v1/notifications/
```
Comment (raneq) [inline .travis.yml:15]: > To instruct Travis to notify Galaxy when a build completes

cool! :D

File: tasks/certificate.yml (hunk 1 above)
Comment (enricostano) [reply tasks/certificate.yml]: Yep, I prefer @raneq 's proposal.

Comment (danypr92) [reply tasks/certificate.yml]: Applied!

**Comment by enricostano** [comment]
@danypr92 could you please add GitHub magic to the description to close the related issue once this PR gets merged? Thanks!

**Comment by raneq** [comment]
will this work? ↓
github! this issue fixes #17 !

edit: nope, only description (maybe titles too?)

**Comment by enricostano** [comment]
We generally just use descriptions.... I don't understand why we stopped doing this.

**Pull request merged**